elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStore.java

```diff
@@ -38,18 +38,14 @@ public MetaDataStore() {
 
     public void populateEntityDictionary(EntityDictionary dictionary) {
         super.populateEntityDictionary(dictionary);
-
-        if (dictionary instanceof AggregationDictionary) {
```

Comment:
This method should bind the metadata classes into the dictionary.

Reply:
super class is doing it